packages/shorebird_cli/pubspec.yaml

```diff
@@ -25,6 +25,7 @@ dependencies:
   meta: ^1.9.0
   path: ^1.8.3
   platform: ^3.1.0
+  plist_parser: ^0.0.9
```

Comment:
We're going to need to do a license review of our dependencies at some point.  lgtm though.

Reply:
Yep, agreed. This particular package has the MIT license, fwiw https://github.com/dirablue/plist_parser/blob/master/LICENSE